dashboard/app/models/gallery_activity.rb

```diff
@@ -24,7 +24,6 @@
 
 class GalleryActivity < ActiveRecord::Base
   belongs_to :user
-  belongs_to :activity
```

Comment:
Note that it probably doesn't make sense to say it belongs to `:user_level`, since the `user_level` may reflect a `level_source` from a prior attempt (so `gallery_activity.level_source_id != user_level.level_source_id`).